lib/AhrefsAPI.php

```diff
@@ -448,64 +459,139 @@ private function getContent($multi = false)
             }
         }
 
-        $mh = curl_multi_init();
-        foreach ($links as $key => $params) {
-            $ch[$key] = curl_init();
-            //setting the links
-            curl_setopt($ch[$key], CURLOPT_URL, $this->apiURL.'/?'.$params.'&token='.$this->token);
-            curl_setopt($ch[$key], CURLOPT_HEADER, 0);
-            if ($this->post) {
-                curl_setopt($ch[$key], CURLOPT_POST, 1);
-                curl_setopt($ch[$key], CURLOPT_HTTPHEADER, array('Content-Type: text/plain'));
-                curl_setopt($ch[$key], CURLOPT_POSTFIELDS, $this->post);
-                curl_setopt($ch[$key], CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_1_0);
-                curl_setopt($ch[$key], CURLOPT_LOW_SPEED_LIMIT, 1);
-                curl_setopt($ch[$key], CURLOPT_LOW_SPEED_TIME, 2400);
-                $this->post = 0;
-            }
-            curl_setopt($ch[$key], CURLOPT_SSL_VERIFYHOST, 0);
-            curl_setopt($ch[$key], CURLOPT_SSL_VERIFYPEER, 0);
-            curl_setopt($ch[$key], CURLOPT_ENCODING, 'gzip,deflate');
-            curl_setopt($ch[$key], CURLOPT_RETURNTRANSFER, 1);
-            curl_setopt($ch[$key], CURLOPT_CONNECTTIMEOUT, 20);
-            curl_setopt($ch[$key], CURLOPT_TIMEOUT, 240); //timeout in seconds
-            curl_multi_add_handle($mh,$ch[$key]);
-        }
+        if ( $this->useGuzzle ) {
```

Comment:
i think we can put the guzzle and curl part in functions to avoid a long `if` block
ie: 
```
if ($this->useGuzzle) {
  $results = $this->getContentGuzzle();
} else { 
  $results = $this->getContentCurl();
}
```